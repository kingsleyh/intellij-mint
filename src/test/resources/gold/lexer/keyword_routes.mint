routes {

  /home {
    sequence {
      Application.setPage("home")
    }
  }

  / {
    sequence {
      Application.setPage("home")
    }
  }

   * {
     Application.setPage("not_found")
   }

}