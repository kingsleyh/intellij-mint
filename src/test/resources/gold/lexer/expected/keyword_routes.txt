routes ('routes')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
/ ('/')
LOWER_IDENT ('home')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
sequence ('sequence')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n      ')
CAPITALISED_IDENT ('Application')
. ('.')
LOWER_IDENT ('setPage')
( ('(')
STRING_LITERAL ('"home"')
) (')')
WHITE_SPACE ('\n    ')
} ('}')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n  ')
/ ('/')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
sequence ('sequence')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n      ')
CAPITALISED_IDENT ('Application')
. ('.')
LOWER_IDENT ('setPage')
( ('(')
STRING_LITERAL ('"home"')
) (')')
WHITE_SPACE ('\n    ')
} ('}')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n   ')
* ('*')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n     ')
CAPITALISED_IDENT ('Application')
. ('.')
LOWER_IDENT ('setPage')
( ('(')
STRING_LITERAL ('"not_found"')
) (')')
WHITE_SPACE ('\n   ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')